build.proj

```diff
@@ -3,8 +3,11 @@
   <PropertyGroup>
     <!-- Capture OSGroup passed to command line for setting default FilterToOSGroup value below -->
     <_OriginalOSGroup>$(OSGroup)</_OriginalOSGroup>
-    <TestOsTarget Condition="'$(OSGroup)'!=''">$(OSGroup)</TestOsTarget>
-    <TestOsTarget Condition="'$(OSGroup)'==''">$(OSEnvironment)</TestOsTarget>
+  </PropertyGroup>
+  <PropertyGroup>
+    <InputOSGroup>$(OSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(FilterToOSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(OS)</InputOSGroup>
```

Comment:
I thought OSEnvironment was specified by the build.props and build.proj was declared before build.props...is that incorrect?